website/datafuse/docs/sqlstatement/other-functions/sleep.md

```diff
@@ -0,0 +1,42 @@
+---
+id: sleep
+title: Sleep
+---
+
+Sleeps 'seconds' seconds on each data block.
```

Comment:
Interesting function, is it used for test under some cases?

Reply:
Yes, for example, this can be very useful when we test graceful shutdown of the server (we always try to wait for the queries to complete and kill the unfinished query after a period of time)